src/test/java/com/dev/bootbasic/user/repository/JdbcUserRepositoryTest.java

```diff
@@ -0,0 +1,58 @@
+package com.dev.bootbasic.user.repository;
+
+import com.dev.bootbasic.user.domain.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.JdbcTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@JdbcTest
+@ActiveProfiles("test")
+@Import(JdbcUserRepository.class)
+public class JdbcUserRepositoryTest {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    private User user;
+
+    @BeforeEach
+    public void setup() {
+        setUser();
+    }
+
+    @Test
+    public void createTest() {
+        User savedUser = userRepository.create(user);
+        assertThat(savedUser).isEqualTo(user);
+    }
+
+    @Test
+    public void findByIdTest() {
+        userRepository.create(user);
+        Optional<User> foundUser = userRepository.findById(user.getId());
+        assertThat(foundUser.get()).isEqualTo(user);
+    }
+
+    @Test
+    public void deleteByIdTest() {
+        userRepository.create(user);
+        userRepository.deleteById(user.getId());
+        Optional<User> foundUser = userRepository.findById(user.getId());
+        Assertions.assertTrue(foundUser.isEmpty());
+    }
+
+    private void setUser() {
+        UUID id = UUID.randomUUID();
+        user = User.of(id, "test-user", LocalDateTime.now());
+    }
```

Comment:
```suggestion
    private User createUser() {
        UUID id = UUID.randomUUID();
        return User.of(id, "test-user", LocalDateTime.now());
    }
```

이렇게 하는게 가독성이 좋을 것 같아요

Reply:
반영했습니다 감사합니다!
[8e0b990](https://github.com/prgrms-be-devcourse/springboot-basic/pull/798/commits/8e0b9902c426e58f8b21beeea3919907600e2249)